src/app/pages/list/status-list/status-list.component.scss

```diff
@@ -45,7 +45,7 @@ section.list-section {
                     margin-top: -8px;
                 }
                 &.mat-expanded {
-                    border-bottom: 1px solid rgba(0, 0, 0, 0.12);
+                    border-bottom: 1px solid rgba(map-get($blui-black, 500), 0.12);
```

Comment:
This seems like a fix that should be applied at the theme-level instead of the design patterns.
Whenever the issue #175 was made, the requested changes don't only apply to the examples within the design patterns repo.  All expansion panels should have their light/dark theme border-bottoms updated.

Reply:
Sure @emclaug2 , I have transferred this issue. The reason I fixed this issue in NG design pattern is here https://github.com/brightlayer-ui/angular-themes/issues/107#issuecomment-1091135279. But right now I have tried this changes it is not effecting drawer. So, fixed issue in angular themes. Thanks!!!

Reply:
Closing this PR. Since issue is transferred to angular-themes